### DISCUSSION TITLE
Audio - survey response

### DISCUSSION THREAD

**Discussion by kassyec** [jspsych/jsPsych#1848] [Q&A]
Hi everyone.
This is my first time coding for a summer research project. I am a bit lost on how to make some things work. 
I need to design an experiment that plays audio clips and then uses a slider or a likert scale to rate the audio. This needs to be done on 3 different dimensions. Is there any way I can change the existing plugins to build an audio-likert-response (audio plays and then on the screen there are 3 different questions), or change the audio-slider-response to allow it to have 3 different questions being displayed?

Thank you guys!

**Comment by jodeleeuw** [comment]
Hi @kassyec,

Do you need the likert options on the screen while the audio is playing?

**Reply by MatthieuFra** [reply]
Hi @jodeleeuw , I'm really interested in having likert options on the screen while audio is playing, is it possible ? 

Thanks!
